src/app/page.js

```diff
@@ -176,6 +198,16 @@ export default function Home() {
                        setOrdinalsPreviewFrame(value);
                      }
                    }} />
+            <CheckList visible={orderData.get('contentType') === ORDERS_TYPE} codeValue={orderData.get('orders')}
```

Comment:
Instead of making this a tab (which is more for creating things), I would put it down at the bottom of the page like Unisat.
<img width="1789" alt="Screenshot 2023-10-19 at 9 06 16 AM" src="https://github.com/thaddeusdiamond/btc-toolkit/assets/648325/612ab5a9-83ae-4371-9c30-de4814109694">